parachain/runtime/src/lib.rs

```diff
@@ -495,13 +500,107 @@ impl Convert<AccountId, [u8; 32]> for AccountId32Convert {
     }
 }
 
+#[cfg(feature = "cumulus-polkadot")]
```

Comment:
Would it be possible to make an inline `mod` with this conditional compilation, and to put all items that now have individual compilation guards inside it?

Reply:
Good idea, I will do this in a subsequent PR as I want to do the renaming of currency types soonish.